aws-logs-metricfilter/pom.xml

```diff
@@ -0,0 +1,209 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>software.amazon.logs.metricfilter</groupId>
+    <artifactId>aws-logs-metricfilter-handler</artifactId>
+    <name>aws-logs-metricfilter-handler</name>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    </properties>
+
+    <dependencies>
+        <!-- https://mvnrepository.com/artifact/software.amazon.cloudformation/aws-cloudformation-rpdk-java-plugin -->
+        <dependency>
+            <groupId>software.amazon.cloudformation</groupId>
+            <artifactId>aws-cloudformation-rpdk-java-plugin</artifactId>
+            <version>1.0.4</version>
```

Comment:
might need to upgrade this if you run into the same issue @xiwhuang ran into: `Resource specification is invalid: Additional properties are not allowed ('resourceLink' was unexpected)`

https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-cloudformation/pull/6#discussion_r421774003

Reply:
Is that error from `cfn validate`? 

`Resource specification for AWS::Logs::MetricFilter is valid`